cudax/include/cuda/experimental/__device/attributes.cuh

```diff
@@ -32,31 +32,8 @@ namespace cuda::experimental
 namespace detail
 {
 
-_CCCL_NODISCARD inline int __get_attr_impl(::cudaDeviceAttr __attr, int __dev)
-{
-  int __value = 0;
-  _CCCL_TRY_CUDA_API(::cudaDeviceGetAttribute, "failed to get device attribute", &__value, __attr, __dev);
-  return __value;
-}
-
-template <::cudaDeviceAttr _Attr>
-struct __dev_attr
-{
-  using type = int;
-
-  _CCCL_NODISCARD constexpr operator ::cudaDeviceAttr() const noexcept
-  {
-    return _Attr;
-  }
```

Comment:
Important: it don't see this conversion anymore after your changs. This is a breaking change.

Reply:
That conversion is still there on line 40, thankfully hidden by the github UI

Reply:
:D